apps/dashboard/.env.example

```
@@ -8,16 +8,30 @@ NEXT_PUBLIC_CLERK_SIGN_UP_URL=/auth/sign-up
 
 # Planetscale
 
+# docker automatically sets it to the required value. However, if you are running the dashboard locally, set it to http://locahost:3900
 DATABASE_HOST="aws.connect.psdb.cloud"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)